.travis.yml

```diff
@@ -1,19 +1,14 @@
-language: node_js
```

Comment:
We might consider upgrading to trusty (https://docs.travis-ci.com/user/trusty-ci-environment/) since we don't need mysql installed anymore